assets/src/data/eventespresso/core/entities.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * Internal dependencies
+ */
+import { getMethodName } from '../base-entities';
+import { MODEL_NAMES } from '../../model';
+
+/**
+ * This method creates selectors for each registered model name wrapping the
+ * generic source selectors.
+ *
+ * @param {Object<function>} source
+ * @return {Object<function>} All the generated selectors.
```

Comment:
It's a jsdoc standard (see http://usejsdoc.org/tags-type.html) but actually I've implemented it wrong here.  The format should be:

```js
@param {Object.<string, function>} source
```

Reply:
Actually in reading more (I knew I came across this syntax somewhere), the syntax I used is considered acceptable (see discussion [here](https://github.com/jsdoc3/jsdoc/issues/426) ) but just not documented with the spec.  PhpStorm understands this syntax (and that's mostly what matters to me).  Usage of the `.` or `<string,value>` is optional.  So my usage here `{Object<function>}` simply means that it is an object where every property value is a function (in this case `string` would be unnecessary because in js property names are always strings for objects).

Reply:
Ya they don't directly reference it but indirectly that some IDE's support Closure docs (which jsdocs is based off of I think).  PhpStorm actually suggested some of these in auto-completes so it looks like PhpStorm supports it (and I haven't seen issues).  We can definitely revisit and do search and replaces later if needed but for now I'm inclined to leave as is.